08-collab.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: page
+title: Version Control with Git
+subtitle: Collaborating
+minutes: 25
+---
+> ## Learning Objectives {.objectives}
+>
+> *   Explain how to collaborate pushing to a common repo
+
+For the next step, get into pairs.
+Pick one of your repositories on GitHub to use for collaboration.
+
+> ## Practicing by yourself {.callout}
+>
+> If you're working through this lesson on your own, you can carry on by opening
+> a second terminal window, and switching to another directory (e.g. `/tmp`).
+> This window will represent your partner, working on another computer. You
+> won't need to give anyone access on GitHub, because both 'partners' are you.
```

Comment:
El 11/05/2015, a las 10:51, Simon Waldman <notifications@github.com> escribió:
> Happy to, but it won't be today. If you're wanting to get it wrapped up, please go ahead and do it :-)
> 
> 
Good, I'll leave to you then, so you get credit for it.